source/whats-new.txt

```diff
@@ -206,6 +207,16 @@ New features of the 4.6 Java driver release include:
   for more information.
 
 .. _version-4.5:
+.. _version-4.5.1:
+
+What's New in 4.5.1
+-------------------
+
+If the DNS server returns an NXDomain error,
+indicating a non-existent domain, the 4.5.1
+driver no longer throws an exception.
```

Comment:
[question]

Non-blocking question, but I'm curious - what does it do? Fail quietly, or does it work as normal and just log a message? This seems like something that should have some type of error message associated.

Reply:
It ignores it. If you look at Jeff's commits, it's literally just an empty catch block for this exception.

I tend to agree with you about handling such things, though I assume that isn't the end of the lookup process and there's other logic to identify the right host address. (I'm also trusting that Jeff et al. know a lot more about this fix than I do...)